universum/configuration_support.py

```diff
@@ -7,15 +8,131 @@
 __all__ = [
     "Step",
     "Configuration",
-    "Variations",
     "set_project_root",
     "get_project_root"
 ]
 
 
 class Step:
     """
-    Step is a collection of configurable project-specific data needed for Universum operations.
+    Step is a collection of configurable project-specific data entries needed for Universum functionality and passed to
```

Comment:
An instance of this class defines a single launch of an external command by Universum. <maybe a link to `configuring.rst` here> It determines execution parameters and result handling.